modules/exploits/windows/http/easy_adress_book_web_server.rb

```diff
@@ -0,0 +1,84 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = ExcellentRanking
```

Comment:
It shouldn't be `ExcellentRanking`. With the module as is `NormalRanking` should be used.

https://github.com/rapid7/metasploit-framework/wiki/Exploit-Ranking